packaging/helm-charts/helm3/strimzi-kafka-operator/crds/040-Crd-kafka.yaml

```diff
@@ -8229,6 +8230,7 @@ spec:
                               - full
                               - add-brokers
                               - remove-brokers
+                              - remove-disks
```

Comment:
I think it's changing because the autorebalance related API class definition is using the same enum to get the modes (so also "full" is included even if not really supported) :-/ And when adding remove-disks, it's there as well.
The only solution I see is not using the enum for autorebalance but just a String and adding additional checks on the values, or just sticking with what we have.

Reply:
Or maybe better, adding a new `KafkaAutoRebalanceMode` with just add-brokers and remove-brokers mode.

Reply:
Yeah, I think you can have a separate Enum with the values supported there.

Reply:
Ok then, I am going to open a trivial PR for this because it's unrelated to this PR. @ShubhamRwt you can then rebase and continue with this one.